.editorconfig

```diff
@@ -0,0 +1,34 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+# WordPress Coding Standards
+# http://make.wordpress.org/core/handbook/coding-standards/
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+tab_width = 4
+indent_style = tab
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
+indent_style = space
+indent_size = 2
+
+[*.txt]
+trim_trailing_whitespace = false
+
+[*.json]
+insert_final_newline = false
+indent_style = space
+indent_size = 2
+
+[*.yml]
```

Comment:
Since this is the same as Markdown, you can change line 18 to `[{*.md,*.yml}]` and delete this block.